edgelet/edgelet-core/src/settings.rs

```diff
@@ -114,13 +114,12 @@ impl ManualDeviceConnectionString {
         let mut device_id = None;
         let mut hub = None;
 
-        let parts: Vec<&str> = self.device_connection_string.split(';').collect();
-        for p in parts {
-            let s: Vec<&str> = p.split('=').collect();
-            match s[0] {
-                SHAREDACCESSKEY_KEY => key = Some(s[1].to_string()),
-                DEVICEID_KEY => device_id = Some(s[1].to_string()),
-                HOSTNAME_KEY => hub = Some(s[1].to_string()),
+        for sections in self.device_connection_string.split(';') {
+            let mut parts = sections.split('=');
```

Comment:
remove mut

Reply:
It looks like I mangled my git branch with an accidental merge. This change is from a commit I did not author: https://github.com/R25l84IHeXjIxy6HO1QXn0y0Dq9mt8EN/iotedge/commit/d0803738c91b325ecddf981dc30d59ef7e4b35ab#diff-14dbbbc5dca73b659a5aafce9e0fa7a1.